addons/framework/statsTracking/fn_statsInitLocal.sqf

```diff
@@ -11,15 +11,17 @@
 
 waitUntil {not isNull player};
 
+Shots_Fired = 0;
+
 if (typeOf player isEqualTo "potato_spectate_playableSpectator" || typeOf player isEqualTo "VirtualCurator_F" || (getText (missionConfigFile >> "cmfVers") isEqualTo "") || (getText (getMissionConfig "Header" >> "gameType") isEqualTo "CMFSPCL")) exitWith {};
 
 [0, 0, (getPlayerUID player), true] remoteExec ["CMF_fnc_updateStatArray", 2];
 
-player setVariable ["Shots_Fired", 0];
+player setVariable ["Player_UID", (getPlayerUID player)];
 
 player addEventHandler ["FiredMan", {
 	params ["_unit", "_weapon", "_muzzle", "_mode", "_ammo", "_magazine", "_projectile", "_vehicle"];
-	[_unit] call CMF_fnc_localShotsFired;
+	[] call CMF_fnc_localShotsFired;
```

Comment:
`_shotsFired = _shotsFired + 1` here.

We cannot be doing calls every time someone fires.